lib/screenplay/places/builder.ex

```diff
@@ -228,6 +234,11 @@ defmodule Screenplay.Places.Builder do
     stop_id
   end
 
+  defp get_stop_id({_id, %{app_params: %Elevator{elevator_id: elevator_id}}}) do
+    {:ok, facility} = @facilities_mod.fetch(elevator_id)
+    get_in(facility, ~w[relationships stop data id])
```

Comment:
Is this allowed to return `nil`? If not, we should fetch the field in some other way or guard against it — `get_in` returns `nil` if any part of the path doesn't exist.

One way to do this would be destructuring:

```elixir
%{"relationships" => %{"stop" => %{"data" => %{"id" => stop_id}}}} = facility
stop_id
```

Reply:
Good catch. I neglected to think about bad data/errors occurrence in the fetch.